physx-sys/src/physx_api.cpp

```diff
@@ -245,13 +273,20 @@ extern "C"
         return &gAllocator;
     }
 
+    // TODO (nises): should this be removed?
```

Comment:
I don't think anything in physx-rs uses it. But it seems to be exported?

Reply:
One can use physx-sys directly, which makes this required.